README.md

```diff
@@ -1,24 +1,33 @@
-# Handlersjs
-## Configure Git
+# Handlers.js
 
-Start by installing Git and configuring your global Git name and email address. You can do so by executing the following commands. Please make sure to use exactly the same name and email address as in your Github profile.
+Handlers.js provides a modular framework based on the abstraction of software components as _handlers_ of data. Using the handlers in this repository, and custom handlers you build yourself, you can compose multiple specialized modules into a full application. Building software this way ensures easily maintainable, configurable and rewireable applications.
 
-```
-$ git config --global user.name "John Doe"
-$ git config --global user.email john@digita.ai
-```
+The real power of — and inspiration for — Handlersjs comes from the [Components.js](https://componentsjs.readthedocs.io) dependency injection framework, for which each package provides the necessary component configuration files. Combining Handlers.js with Components.js enables you to define each software component separately, and then wire them together declaratively using semantic configuration in JSON files, instead of hard-wiring them together in code.
```

Comment:
```suggestion
The real power of — and inspiration for — Handlers.js comes from the [Components.js](https://componentsjs.readthedocs.io) dependency injection framework, for which each package provides the necessary component configuration files. Combining Handlers.js with Components.js enables you to define each software component separately, and then wire them together declaratively using semantic configuration in JSON files, instead of hard-wiring them together in code.
```